apps/frontend/src/components/global/ExportSplunkModal.vue

```diff
@@ -45,6 +45,11 @@
                   Are you sure that you have configured it properly?
                 </span>
                 <br />
+                <span
+                  >The 'index' field requires a valid index name from the
+                  specified Splunk instance.</span
+                >
+                <br />
                 <span>
                   For installation instructions and further information, check
```

Comment:
<img width="559" alt="image" src="https://github.com/mitre/heimdall2/assets/2015821/1da296ba-c757-4460-b34e-494302533653">

that leads out to a separate plugin we had previously written, but i thought we were using the splunk mapper at this point?

Reply:
we might want to harvest the accreditations and material from the readme and put it into a page on the heimdall wiki if i'm correct about us using hdf converters now so that we don't lose track of like those screenshots and other explanatory material (so long as that material is accurate)